indico/web/templates/forms/markdown_widget.html

```diff
@@ -39,19 +39,9 @@
 
             <script type="text/html" class="markdown-info-text">
                 <p>
-                    {% trans -%}
-                        Markdown is a lightweight markup language that makes it easier to write rich text content.
-                        Here are some examples:
-                    {%- endtrans %}
-                </p>
-                <ul>
-                    <li><span class="mono">*apples*</span> - <em>apples</em></li>
-                    <li><span class="mono">**oranges**</span> - <strong>oranges</strong></li>
-                    <li><span class="mono">[Google][https://www.google.com]</span> - <a href="http://www.google.com">Google</a></li>
-                </ul>
-                <p>
-                    {% trans link='<a href="https://daringfireball.net/projects/markdown/syntax">'|safe, link_end='</a>'|safe -%}
-                        Check the icon bar for more options, or consult the {{ link }}language reference{{ link_end }}.
+                    {% trans link='<a href="http://commonmark.org/help/">'|safe, link_end='</a>'|safe -%}
```

Comment:
How about `link_start` instead of `link`, since you have `link_end`? :smile: 
Or maybe `link_open` and `link_close`, since you "open" and "close" the tag?
But anyhow, not important!